fava/static/javascript/main.js

```diff
@@ -102,6 +106,7 @@ function doPoll() {
       if (data.changed) {
         $('#reload-page').classList.remove('hidden');
         e.trigger('file-modified');
+        e.trigger('warning', _('Beancount file changed outside of Fava. Reload is required.'));
```

Comment:
The first sentence is not entirely correct e.g. a document could have been added. Also, "required" sounds a bit harsh ;) Maybe (the "click to reload" would be nice IMHO):
> File change detected. Click to reload.